### ISSUE TITLE
Feature request: add 'squash commits before merge' option

### ISSUE THREAD

**Issue by b1rdex** [smarkets/marge-bot#152]
Would be great to be able to allow to set merge request squash option to true/false.

**Comment by micheelengronne** [comment]
Indeed. It would be cool to have a cleaner commit history and respect Angular Commit message conventions for instance.

**Comment by bensampaio** [comment]
Yeah would love to see this implemented. What are the reasons for the current behavior?
